doc/dev/developer_guide/running-tests-locally.rst

```diff
@@ -50,17 +52,12 @@ Running your first test
 The Python tests in Ceph repository can be executed on your local machine
 using `vstart_runner.py`_. To do that, you'd need `teuthology`_ installed::
 
-    $ git clone https://github.com/ceph/teuthology
-    $ cd teuthology/
-    $ virtualenv -p python2.7 ./venv
+    $ virtualenv --python=python3 venv
```

Comment:
@tchaikov For consistency, can you also update this line
https://github.com/ceph/ceph/blob/master/src/pybind/mgr/dashboard/run-backend-api-tests.sh#L44
to sth like:
```
virtualenv --python=${TEUTHOLOGY_PYTHON_BIN:-/usr/bin/python3} venv
```

Reply:
sure. updated and repushed.